src/ICeWall/doc/set_bump_options.md

```diff
@@ -0,0 +1,60 @@
+## set_bump_options
+
+### Synopsis
+```
+  % set_bump_options \
+        [-pitch pitch] \
+        [-spacing_to_edge spacing] \
+        [-bump_pin_name pin_name] \
+        [-cell_name bump_cell_table] \
```

Comment:
The documentation says this can be a single cell name, but that generates an error.

Reply:
Updated so that it now works with a single cell name